.github/workflows/windows.yml

```diff
@@ -413,66 +413,136 @@ jobs:
           cmake --build bld --config '${{ matrix.type }}' --target test-ci
 
   msvc:
-    name: 'msvc (${{ matrix.arch }}, ${{ matrix.plat }}, ${{ matrix.config }})'
+    name: 'msvc vcpkg (${{ matrix.name }})'
     runs-on: windows-latest
     timeout-minutes: 30
     strategy:
       matrix:
         include:
-          - { arch: 'x64', plat: 'windows', tflags: '~1516 ~2301 ~2302 ~2303 ~2307', config: '-DENABLE_DEBUG=ON -DCURL_USE_SCHANNEL=ON  -DBUILD_SHARED_LIBS=OFF -DENABLE_UNICODE=ON ', type: 'Debug' }
-          - { arch: 'x64', plat: 'windows', tflags: '~1516 ~2301 ~2302 ~2303 ~2307', config: '-DENABLE_DEBUG=ON -DCURL_USE_SCHANNEL=OFF -DBUILD_SHARED_LIBS=OFF -DENABLE_UNICODE=OFF', type: 'Debug' }
-          - { arch: 'x64', plat: 'windows', tflags: '~1516 ~2301 ~2302 ~2303 ~2307', config: '-DENABLE_DEBUG=ON -DCURL_USE_SCHANNEL=OFF -DHTTP_ONLY=ON          -DENABLE_UNICODE=OFF', type: 'Debug' }
+        # name should be unique and without specail chars. It create a folder with that name for cache the libreries.
+        - name: schannel
+          install: nghttp2 zstd libssh2[openssl]
```

Comment:
Perhaps use libssh2-schannel instead? It seems like a waste to install openssl just for the sake of libssh2. Also mixing crypto backends is best to avoid.

Reply:
on vcpkg thereisn't libssh2 with schannel. only openssl.

Reply:
I can remove the libssh2 from schannel build.

Reply:
That's a bummer. Agreeing to remove libssh2 then. It's tested in the other job anyway.

Reply:
Are you sure the libssh2 that you use in other places (like linux mingw osx) not coming with openssl?
I think you don't aware of it.

Reply:
@vszakats libssh2 come only with openssl.
https://github.com/libssh2/libssh2/blob/master/CMakeLists.txt

Reply:
libssh2: not exactly, it also supports WinCNG (which is the native Win32 crypto API), mbedTLS and libgcrypt.

I think it's enough to test libssh2 with openssl here. An additional Schannel + libssh2/openssl build doesn't add any further extra coverage for the build or tests, like a libssh2/WinCNG build would.

Reply:
Thank you for sharing with me this information!

Reply:
Actually I check on vcpkg, there is an option with WinCNG backend. I will add it to Schannel.

Reply:
I added.

Reply:
Nice, thanks!